README.md

```diff
@@ -161,31 +161,24 @@ Output is the `CameraPoses.csv` file, in which the first line contains column na
 | Column | Type  | Name | Description                |
 | ------ | ----- | ---- | -------------------------- |
 | 1      | int   | id   | 0-indexed frame id         |
-| 2      | float | tx   | X position in meters       |
-| 3      | float | ty   | Y position in meters       |
-| 4      | float | tz   | Z position in meters       |
-| 5      | float | qw   | Rotation quaternion W      |
-| 6      | float | qx   | Rotation quaternion X      |
-| 7      | float | qy   | Rotation quaternion Y      |
-| 8      | float | qz   | Rotation quaternion Z      |
+| 2      | float | tx   | X position in centimeters  |
+| 3      | float | ty   | Y position in centimeters  |
+| 4      | float | tz   | Z position in centimeters  |
+| 5      | float | qx   | Rotation quaternion X      |
+| 6      | float | qy   | Rotation quaternion Y      |
+| 7      | float | qz   | Rotation quaternion Z      |
+| 8      | float | qw   | Rotation quaternion W      |
 | 9      | float | t    | Timestamp in seconds       |
 
-The coordinate system for saving camera positions and rotation quaternions is a right-handed coordinate system. When looking through a camera with zero rotation in the target coordinate system:
-- X axis points to the right
-- Y axis points down
-- Z axis points straight away from the camera
-
-<img src="ReadmeContent/OutputCoordinateSystem.png" alt="Output coordinate system" width="250" style="margin:10px"/>
-
-Note that this differs from Unreal Engine, which internally uses the left-handed Z-up coordinate system.
+The coordinate system for saving camera positions and rotation quaternions is the same one used by Unreal Engine, a left-handed Z-up coordinate system. This system is pretty much only used by UE and will most likely require a conversion for your later use. Still, it seems to be the cleanest option, as exported values will match the numbers displayed inside the engine.
```

Comment:
> The coordinate system for saving camera positions and rotation quaternions is the same one used by Unreal Engine, a ***left-handed*** Z-up coordinate system.

Coordinates will ***likely require conversion*** to more common reference frames for typical computer vision applications. For more information we recommend [this Reddit post](https://www.reddit.com/r/gamedev/comments/7qh3sa/a_coordinate_system_chart_of_different_engines/).